pkg/controller/persistentvolume/persistentvolume_controller.go

```diff
@@ -484,7 +482,164 @@ func (ctrl *PersistentVolumeController) syncBoundClaim(claim *api.PersistentVolu
 func (ctrl *PersistentVolumeController) syncVolume(volume *api.PersistentVolume) error {
 	glog.V(4).Infof("synchronizing PersistentVolume[%s]: %s", volume.Name, getVolumeStatusForLogging(volume))
 
-	return nil
+	// [Unit test set 4]
```

Comment:
names are more durable than numbers, but it's minor for now.  Do not block this PR on it.